website/docs/r/cert_request.html.md

```diff
@@ -79,6 +79,13 @@ corresponding to the similarly-named attributes defined in
 
 * `serial_number` (string)
 
+-> **Note:** Versions of this provider compiled by Go 1.9.x or older may
```

Comment:
Could you edit this to reference the provider version, instead of the go version? Most users won't (and shouldn't) know what version of Go a given provider was compiled with. 

You can reference the upcoming version 1.2.0:

```Versions of this provider prior to 1.2.0 may ... ```


Alternatively, if you would like to give maintainers of this repository access to your fork, I can push this change for you and get it merged (https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/).

Thanks!

Reply:
Hey @mildwonkey I pushed another patchset with the suggested change. Allow edits from maintainers is already checked so if you need to make additional edits I think you should be able to, otherwise I'm happy to make them myself.